doc/rst/platforms/infiniband.rst

```diff
@@ -105,24 +105,27 @@ Selecting a Spawner
 Under the covers, ``gasnetrun_ibv``-based launchers must figure out
 how to spawn jobs and get them up and running on the compute nodes.
 GASNet's three ways of doing this on InfiniBand systems are ``ssh``,
-``pmi``, and ``mpi``.  When MPI is detected at configure time, it will
-be selected as the default, but we recommend using one of the other
-options if possible.
-
-* ``ssh``: Based on our experience, we recommend launching with the
-  ``ssh`` option, if possible.  This requires the ability to ``ssh``
-  to the system's compute nodes, which is not supported by all
-  systems, depending on how they are configured.  See the following
-  sub-section for details on this option.
+``pmi``, and ``mpi``, described just below.  When MPI is detected at
+configure time, it will be selected as the default, but we recommend
+using one of the other options if possible.  This can be done by
+setting the ``GASNET_IBV_SPAWNER`` environment variable, whose legal
+values are:
+
+* ``ssh``: Based on our experience, this is the preferred option, when
+  possible.  This requires the ability to ``ssh`` to the system's
+  compute nodes, which is not supported by all systems, depending on
+  how they are configured.  See the following sub-section for details
+  on this option.
   
 * ``pmi``: When GASNet's configure step detects a PMI-capable job
   scheduler like Slurm, ``pmi`` can be the next best choice because it
   often "just works" and can reduce overhead compared to ``mpi``.
```

Comment:
Might be worth a link to https://bitbucket.org/berkeleylab/gasnet/src/master/other/pmi-spawner/README as part of a "call out" that the environment variable `PMIRUN_CMD` is used to configure how jobs are started; or just as "for more information".
Same for the omnipath document.

Reply:
Here, I've added a final sentence:

> For more information about this option, including how to configure job launch via ``PMIRUN_CMD``, see the [GASNet README for the PMI-based spawner](https://bitbucket.org/berkeleylab/gasnet/src/master/other/pmi-spawner/README).

Reply:
With a bit more thought, I'm also following each of these up with the path within the Chapel source tree where the files live, for users who may not have www access.  E.g., 

> see the [GASNet README for the PMI-based spawner (https://bitbucket.org/berkeleylab/gasnet/src/master/other/pmi-spawner/README) (also available at ``$CHPL_HOME/third-party/gasnet/gasnet-src/other/pmi-spawner/README``).